src/Features/CSharp/Portable/ChangeSignature/CSharpChangeSignatureService.cs

```diff
@@ -25,6 +28,8 @@ namespace Microsoft.CodeAnalysis.CSharp.ChangeSignature
     [ExportLanguageService(typeof(AbstractChangeSignatureService), LanguageNames.CSharp), Shared]
     internal sealed class CSharpChangeSignatureService : AbstractChangeSignatureService
     {
+        protected override SyntaxGenerator Generator => CSharpSyntaxGenerator.Instance;
```

Comment:
There's a few methods that use the Generator, and it was simpler to encode it this way rather than always needing to thread through the document / generator.